targets/nuttx-stm32f4/README.md

```diff
@@ -16,6 +16,8 @@ git clone https://github.com/jerryscript-project/jerryscript.git
 git clone https://bitbucket.org/nuttx/nuttx.git
 git clone https://bitbucket.org/nuttx/apps.git
 git clone https://github.com/texane/stlink.git
+cd stlink
+git checkout 3d24377c2878fc328fb09b67cdfbda529d209417
```

Comment:
@akosthekiss Okey, I agree with that. What do you want, just update the changed path for the binary, and that's all?